### PR TITLE
[Android] Replace deprecated Html.fromHtml(String) call

### PR DESCRIPTION
**Pull request by joseluismarti**
## Description
`Html.fromHtml(String)` method was deprecated in API level 24.

`Html.fromHtml` now requires an additional parameter, named `flags`. This flag gives more control about how the HTML is displayed.

## Types of change
- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [X] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:
- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Review by fuzzard — COMMENTED** [review]
My gut feeling is this should just be done inline without the extra function.

**Comment by joseluismarti** [comment]
Alright, I move it to inline

File: tools/android/packaging/xbmc/src/Splash.java.in [hunk 1]
```diff
@@ -537,7 +537,11 @@ public class Splash extends Activity
     AlertDialog.Builder builder = new AlertDialog.Builder(act);
     builder.setTitle(title);
     builder.setIcon(android.R.drawable.ic_dialog_alert);
-    builder.setMessage(Html.fromHtml(message));
+    if (Build.VERSION.SDK_INT >= 24) {
+      builder.setMessage(Html.fromHtml(message, Html.FROM_HTML_MODE_LEGACY));
```
Comment (enen92) [inline tools/android/packaging/xbmc/src/Splash.java.in:541]: This seems to be correct however looks like it was added solely for compatibility reasons. Isn't there a new flag (more explicit) that describes better what we want to do here without something so generic as "legacy"?
I.e. at to what point this flag won't be also deprecated in some newer API revision?

Comment (joseluismarti) [reply tools/android/packaging/xbmc/src/Splash.java.in:541]: Yes, there are more flags, however the `MODE_LEGACY` flag provides the most basic behavior and is identical to the method it replaces. It should not be deprecated in the near future.

**Review by enen92 — APPROVED** [review]

**Pull request merged**